src/components/Edit.js

```diff
@@ -43,6 +44,7 @@ const stateStatuses = {
 const SAVE_TIMEOUT_MS = 1000 * 10
 
 const stateColors = {
+  [stateStatuses.NEEDS_SAVING]: '#e67e22',
```

Comment:
maybe color should be different? kinda 50/50 on this since both mean "dont close the window" in practice